pkg/config/apm.go

```diff
@@ -41,6 +41,7 @@ func setupAPM(config Config) {
 	config.SetKnown("apm_config.log_throttling")
 	config.SetKnown("apm_config.bucket_size_seconds")
 	config.SetKnown("apm_config.watchdog_check_delay")
+	config.SetKnown("apm_config.sync_flushing")
```

Comment:
Are we going to want to allow this to be set by means of an env. var. ? If so, we might want to bind it here in the same function to `DD_APM_SYNC_FLUSHING`, or `DD_APM_LAMBDA`, whichever suits the use case more.